lib/react_client/react_interop.dart

```diff
@@ -105,7 +108,7 @@ class JsRef {
 ///
 /// See: <https://reactjs.org/docs/forwarding-refs.html>.
 ReactJsComponentFactoryProxy forwardRef(Function(Map props, Ref ref) wrapperFunction) {
-  var hoc = _jsForwardRef(allowInterop((JsMap props, JsRef ref) {
+  var hoc = React.forwardRef(allowInterop((JsMap props, JsRef ref) {
```

Comment:
I like this better. @greglittlefield-wf do you see any possible issues here?

Reply:
Nope, looks good to me!